index.html

```diff
@@ -60,68 +60,16 @@ <h1>This be Adam's box.</h1>
-            <h1>Luke Siecinski</h1>
-            <p>My name is Luke and I am learning how to be a web-developer!</p>
-            <h3>Contact info:</h3>
-            <ul>
-                <li>luke24ski@aol.com</li>
-                <li>
-                    <a href="https://www.facebook.com/luke.siecinski">Facebook</a>
-                </li>
-                <li>
-                    <a href="https://twitter.com/lucas_ski24">Twitter</a>
-                </li>
-            </ul>
-        </div>
-        <div>
-            <h1>Kyle Jennings</h1>
-            <img src="http://res.cloudinary.com/kodesign/image/upload/v1516990751/mobilebanner350_s7b12m.jpg" alt="Me">
-            <p> "Horsepower corrupts, absolute horsepower corrupts absolutely - Small block Chevy Machiavelli" </p>
-            <p> Email: Kyleojennings@gmail.com</p>
-        </div>
-        <div>
-            <h1>Areanna Whittingotn</h1>
-            <img src="https://avatars1.githubusercontent.com/u/40213998?s=460&v=4My" alt="thats my face">
-            <h2>Quotes</h2>
-            <p>Just a girl trying to be successful</p>
-            <p>To be beautiful means to be yourself. You don't need to be accepted by others. You need to accept yourself</p>
-            <ol>
-                <li>
-                        <a href="https://www.facebook.com/areyscreations">Facebook</a>
-                </li>
-                <li>
-                    <a href="https://www.instagram.com/areyscreations/">Instagram</a>
-                </li>
-                <li>
-                    <a href="http://www.areys.online">Website</a>
-                </li>
-            </ol>
-        </div>
-        <div>
-            <h1>
-                Welcome to Div 11!
-            </h1>
-            <h2>
-                design by Marcus Boyd
-            </h2>
-            <img src="https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/12144868_10153576916107040_1719552211906372501_n.jpg?_nc_cat=0&oh=186df30e8d3e798a75a7f1cead0ea916&oe=5BAD6ABD" alt="">
-            <p>"People without the knowledge of their past history, origin and culture is like a tree without roots"
-                -Marcus Garvey
-            </p>
-            <ul>
-                <h1>
-                    Contact Info!
-                </h1>
-                <li>
-                    <a href="https://www.facebook.com/foto.persona.9">Facebook</a>
-                </li>
-                <li>
-                    <a href="https://github.com/mboyd3813">github</a>
-                </li>
-            </ul>
+        <div>EIGHT</div>
+        <div class = "kyle"> 
+               <h1>Kyle Jennings</h1>
+               <img src="http://res.cloudinary.com/kodesign/image/upload/v1516990751/mobilebanner350_s7b12m.jpg" alt="Me">
+              <p> "Horsepower corrupts, absolute horsepower corrupts absolutely - Small block Chevy Machiavelli" </p>
+              <p> Email: Kyleojennings@gmail.com</p>
+
         </div>
+        <div>TEN</div>
```

Comment:
Make sure you aren't adding the `NINE`, `TEN`, and `ELEVEN` boxes back into the codebase!